specification/app/resource-manager/Microsoft.App/preview/2024-08-02-preview/CommonDefinitions.json

```diff
@@ -1614,6 +1614,233 @@
         }
       }
     },
+    "ManagedHttpRoutePatch": {
```

Comment:
nit: Could you create a separate `ManagedEnvironment_HttpRoute.json`.

Reply:
can we keep in same file for consistency ? Managed Certs as well as other environment level resource are in the same file and it'd be great if we had this one file for all environment level resources. Thoughts ?

Reply:
Created new file. Thank you for the comment it makes it much easier to follow changes now that there is only one file :)